CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
+| `Features`           | Bullet list that sumarrizes the availbale functionality of the packaged application |
+| `Package Parameters` | Bullet list of the available package parameters that can be used when installing the package, this section can be omitted if there is no package parameters |
+| `Notes`              | Bullet list with any special information to the user about the software and/or package that they need to know about |
 
-Unless the package installer/executable/archive needs some special handling (like the need to read version from file, or something else not available to be handled by AU), anything bigger then few MB should never be downloaded within `au_GetLatest` function. Normally, the files are downloaded during the embeding process by the `au_BeforeUpdate`.
+#### Maintainers
 
-### 1.3.3 Specify correct `NuspecVersion`
+Maintainers of the package in our case are users that have edited a package, either by being the main maintainer of the package or contributed one or more changes to the package.
+The metadata file should always include both current and historical maintainers of the package, if you are editing a package make sure to add yourself as the last maintainer colon delimeted list.
+If the user `chocolatey-community` is not already specified as the maintainer of the package, this user must be added as the first maintainer in the `owners` element.
+One or more maintainers of the maintainers listed in the `owners` element should be equal to a similar name listed in our `CODE_OWNERS` file, and will be considered the main maintainer of the package.
 
-Since commit message is used to push packages to the community repository (CR), prior to commit of the AU package specify correct `NuspecVersion` in the nuspec file as follows:
+#### Icons
 
-1. If you want to fix the exisiting CR package version for which no update is available at vendor specify the `NuspecVersion` the same as `RemoteVersion`.
-1. If you want to push new package version to CR and `RemoteVersion` doesn't exist on CR, set `NuspecVersion` to `0.0`. AU updater will then bring the version to correct one during the next AU run and commit it to the git repository.
-1. If you want to fix package that is pushed but not yet approved (for example it fails verifier) do the same as in first case but request in PR comment that you want version explicitelly set.
+A package must have an icon available if the packaged software has an icon. This icon must be named the same as the package and is placed in the [icons](https://github.com/chocolatey-community/chocolatey-packages/tree/master/icons) directory.
+If the package name ends with either `.install`, `.portable` those suffixes may be ignored in the icon name.
+When an icon is added to this folder with the correct name, it wwill **automatically** be set in the metadata file and the Readme file when our build server updates the package if the metadata file contains an `<IconUrl>` element.
```

Comment:
```suggestion
When an icon is added to this folder with the correct name, it will **automatically** be set in the metadata file and the Readme file when our build server updates the package, if the metadata file contains an `<IconUrl>` element.
```